packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -305,11 +331,13 @@ class Popover extends React.Component {
             min-height: 16px;
             min-width: 16px;
             outline: 0;
-            overflow-x: hidden;
-            overflow-y: auto;
             position: absolute;
             z-index: ${theme.zIndex.middle};
           }
+
+          .popover::webkit-scrollbar {
+            display: none;
+          }
```

Comment:
ahh the scrollbar fix? :D

Reply:
The main fix is in the overflow css changes above, this is just a final check to make sure as the scrollbar width is not included when the calculations are run. Long story short.... this **should** never happen xD